docs/configuration/configuration.md

```diff
@@ -415,6 +415,9 @@ See below for the configuration options for EC2 discovery:
 # The AWS Region.
 region: <string>
 
+# Custom endpoint to be used.
```

Comment:
What happens if both endpoint and region is specified?

Reply:
A custom endpoint has also regions. 

The region needs to be specified regardless of the endpoint to be used. In case of the AWS endpoints you have a set of predefined regions while with a custom endpoint you have the ability to specify any string